PivotGrid: update Integration with Chart (#6964)
DevExpress/devextreme-documentation: concepts/05 UI Components/PivotGrid/150 Integration with Chart/10 Integration with Chart.md

> PivotGrid handles large numeric datasets, which can be difficult to interpret. For clearer data visualization, use the [Chart](/api-reference/10%20UI%20Components/dxChart '/Documentation/ApiReference/UI_Components/dxChart/') component. To link PivotGrid with Chart, call the [bindChart(chart, integrationOptions)](/api-reference/10%20UI%20Components/dxPivotGrid/3%20Methods/bindChart(chart_integrationOptions).md '/Documentation/ApiReference/UI_Components/dxPivotGrid/Methods/#bindChartchart_integrationOptions') method. This article explores the features of this method.

Reviewer: ```suggestion
PivotGrid handles large numeric datasets that can be difficult to interpret. For more detailed data visualization, use the [Chart](/api-reference/10%20UI%20Components/dxChart '/Documentation/ApiReference/UI_Components/dxChart/') component. To link PivotGrid with Chart, call the [bindChart(chart, integrationOptions)](/api-reference/10%20UI%20Components/dxPivotGrid/3%20Methods/bindChart(chart_integrationOptions).md '/Documentation/ApiReference/UI_Components/dxPivotGrid/Methods/#bindChartchart_integrationOptions') method. This article explores the features of this method.
```
